workflows/HiC/defaults.yaml

```diff
@@ -0,0 +1,64 @@
+################################################################################
+# This file is the default configuration of the HiC workflow!
+#
+# In order to adjust some parameters, please either use the wrapper script
+# (eg. /path/to/snakemake_workflows/workflows/HiC/HiC)
+# or save a copy of this file, modify necessary parameters and then provide
+# this file to the wrapper or snakmake via '--configfile' option
+# (see below how to call the snakefile directly)
+#
+# Own parameters will be loaded during snakefile executiuon as well and hence
+# can be used in new/extended snakemake rules!
+################################################################################
+## General/Snakemake parameters, only used/set by wrapper or in Snakemake cmdl, but not in Snakefile
+outdir:
+configfile:
+local: False
+max_jobs: 5
+snakemake_options:
+tempdir: /data/extended/
+## directory with fastq files
+indir:
+## preconfigured target genomes (mm9,mm10,dm3,...) , see /path/to/snakemake_workflows/shared/organisms/
+## Value can be also path to your own genome config file!
+genome:
+## FASTQ file extension (default: ".fastq.gz")
+ext: .fastq.gz
+## paired-end read name extension (default: ['_R1', "_R2"])
+reads: [_R1, _R2]
+mapping_prg: BWA
```

Comment:
What if bowtie is used? The parameters no longer apply. For bowtie we need --reorder. I think we need to allow only one aligner.

Reply:
at this moment only one aligner is used (BWA) so this variable is useless and can be removed